modules/ROOT/pages/get-started.adoc

```diff
@@ -1,6 +1,6 @@
 = Getting Started with Anypoint API Governance
 
-After you become familiar with the UI and concepts covered in the xref:index.adoc[Anypoint API Governance Overview], you are ready to get started governing your APIs.
+After you become familiar with the UI and concepts covered in the xref:index.adoc[Anypoint API Governance Overview], you're ready to get started governing your APIs.
```

Comment:
It might be a subtle difference, but it oversimplifies by making it sound like you start immediately. There are quite a few steps. I'll mull it over. I have a bit of deja vu on this one. This isn't new text. :)